development/tests/regression/run.py

```diff
@@ -66,7 +66,11 @@ def check_vault(num_test):
 
         init_dict, expected_df = test
 
+        init_dict["PARAMETERS"]["const_p"] = init_dict["PARAMETERS"].pop("theta_p")
```

Comment:
We want to be sure to clean this up before the final merge.

Reply:
To be addressed after update of regression test vault

Reply:
Done.